### PR TITLE
Two separate changes

### PR DESCRIPTION
**Pull request by vmurali**
1) Specifiying the sign of the remainder in signed division
2) Explaining the effect of reducing MXLEN on integer register values, PC and CSRs.

### CODE REVIEW

File: src/m.tex [hunk 1]
```diff
@@ -99,7 +99,20 @@ \section{Division Operations}
 DIV and DIVU perform an XLEN bits by XLEN bits signed and unsigned integer
 division of {\em rs1} by {\em rs2}, rounding towards zero.
 REM and REMU provide the remainder of the
-corresponding division operation.  If both the quotient and remainder
+corresponding division operation.
+For signed division, the sign of the remainder and the value of the quotient
+follow the usual conventions: the sign of the remainder is the sign of the
+dividend and the absolute value of the quotient is the same as the quotient for
```
Comment (aswaterman) [inline src/m.tex]: I believe the remark about the absolute value of the quotient is redundant with the previous sentence ("rounding towards zero").  Agreed?

Comment (aswaterman) [reply src/m.tex]: Could move it to the commentary you added below (e.g. "Rounding towards zero implies that the abs val of the quotient is the same as [...].  In combination with the sign of the remainder, this ensures that [...]")

File: src/machine.tex [hunk 2]
```diff
@@ -88,6 +88,19 @@ \subsection{Machine ISA Register {\tt misa}}
 zero after two shifts, then the machine is RV64, else RV128.
 \end{commentary}
 
+\begin{commentary}
```
Comment (aswaterman) [inline src/machine.tex]: Did you determine that a commentary block is sufficient for this?  i.e. do we not need normative text explaining MXLEN's effect on the PC?

File: src/m.tex (hunk 1 above)
Comment (aswaterman) [reply src/m.tex]: I convinced myself that the quotient definition follows from the rounding scheme.  I'm committing a change that includes the sign-of-REM clarification and the commentary below.

File: src/machine.tex (hunk 2 above)
Comment (aswaterman) [reply src/machine.tex]: I think we need new normative text about the PC.  Working out the language with JohnH.

**Comment by vmurali** [comment]
Sorry 😂 I will check the commit when I get a chance this weekend

**Pull request closed without merging**